### PR TITLE
Archived tasks cards [SCI-5274] [SCI-5275]

### PR DESCRIPTION
**Pull request by urbanrotnik**
Jira ticket: [SCI-5274](https://biosistemika.atlassian.net/browse/SCI-5274)

### CODE REVIEW

File: app/assets/javascripts/my_modules/archived.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+(function() {
+  let selectedTasks = [];
+
+  $('#module-archive').on('click', '.restore-button-container', function(e) {
+    e.preventDefault();
+    alert('woop! Not implemented yet!')
```
Comment (aignatov-bio) [inline app/assets/javascripts/my_modules/archived.js:6]: What was that?

File: app/assets/javascripts/my_modules/archived.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  $('#module-archive').on('click', '.task-selector', function() {
+    let taskId = $(this).closest('.task-selector-container').data('task-id');
+    let index = $.inArray(taskId, selectedTasks);
+    let resoteTasksButton = $('.restore-button-container');
+
+    // If checkbox is checked and row ID is not in list of selected folder IDs
+    if (this.checked && index === -1) {
+      selectedTasks.push(taskId);
+      resoteTasksButton.collapse('show');
```
Comment (aignatov-bio) [inline app/assets/javascripts/my_modules/archived.js:17]: just - `.show() / .hide()`

File: app/assets/javascripts/projects/canvas.js.erb [hunk 3]
```diff
@@ -2953,5 +2962,9 @@ var preventCanvasReloadOnSave = (function() {
     })
   }
 
+  function initArchivedView() {
+
```
Comment (aignatov-bio) [inline app/assets/javascripts/projects/canvas.js.erb:2966]: ?

File: app/assets/stylesheets/my_modules.scss [hunk 4]
```diff
@@ -188,3 +188,81 @@
...
+  }
+
+  .panel-default {
+    border: 0;
+    box-shadow: unset;
+    position: unset;
+    width: unset;
+
+    .task-selector-container {
+      padding: 11px 10px 0 16px;
```
Comment (aignatov-bio) [inline app/assets/stylesheets/my_modules.scss:219]: em?

File: app/assets/stylesheets/my_modules.scss [hunk 5]
```diff
@@ -188,3 +188,81 @@
...
+        .date-value {
+          color: $color-black;
+        }
+      }
+    }
+
+    .panel-footer-scinote {
+      background: $color-concrete;
+      border-color: $color-white;
+      height: 34px;
```
Comment (aignatov-bio) [inline app/assets/stylesheets/my_modules.scss:252]: em?

File: app/assets/stylesheets/my_modules.scss [hunk 6]
```diff
@@ -188,3 +188,81 @@
...
+      .nav {
+        border: 0;
+      }
+
+      .tab-content {
+        background-color: $color-concrete;
+        box-shadow: $flyout-shadow;
+
+        .tab-pane > .text-center {
+          margin-top: 5px;
```
Comment (aignatov-bio) [inline app/assets/stylesheets/my_modules.scss:263]: em?

**Review by aignatov-bio — CHANGES_REQUESTED** [review]

File: app/assets/javascripts/my_modules/archived.js (hunk 1 above)
Comment (urbanrotnik) [reply app/assets/javascripts/my_modules/archived.js:6]: It is a placeholder for someone who is doing on this feature. Not part of this task

File: app/assets/javascripts/my_modules/archived.js (hunk 2 above)
Comment (urbanrotnik) [reply app/assets/javascripts/my_modules/archived.js:17]: It's not smooth BS collapse animations. It jumps as a toggling display property

File: app/assets/stylesheets/my_modules.scss (hunk 4 above)
Comment (urbanrotnik) [reply app/assets/stylesheets/my_modules.scss:219]: yes, maybe here

File: app/assets/javascripts/projects/canvas.js.erb (hunk 3 above)
Comment (urbanrotnik) [reply app/assets/javascripts/projects/canvas.js.erb]: removed

**Review by aignatov-bio — APPROVED** [review]

**Pull request merged**